joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/EnumTests.scala

```diff
@@ -48,26 +49,27 @@ class EnumTests extends CCodeToCpgSuite(fileSuffix = FileDefaults.CPP_EXT) {
         |    green = 20,
         |    blue
         |} C;""".stripMargin)
-      inside(cpg.typeDecl.nameNot("<global>").filter(x => !x.isExternal).l) { case List(color, c) =>
-        color.name shouldBe "color"
-        color.code should startWith("typedef enum color")
-        color.aliasTypeFullName shouldBe None
-        c.name shouldBe "C"
-        c.aliasTypeFullName shouldBe Some("color")
-        inside(color.astChildren.isMember.l) { case List(red, yellow, green, blue) =>
-          red.name shouldBe "red"
-          yellow.name shouldBe "yellow"
-          green.name shouldBe "green"
-          blue.name shouldBe "blue"
-        }
-        inside(color.astChildren.isMethod.l) { case List(sinit) =>
-          sinit.name shouldBe "<sinit>"
-          sinit.fullName shouldBe "color:<sinit>"
-          sinit.code shouldBe "green = 20"
-          inside(sinit.ast.isCall.l) { case List(greenInit) =>
-            greenInit.code shouldBe "green = 20"
+      inside(cpg.typeDecl.nameNot(NamespaceTraversal.globalNamespaceName).filter(x => !x.isExternal).l) {
```

Comment:
```suggestion
      inside(cpg.typeDecl.nameNot(NamespaceTraversal.globalNamespaceName).internal.l) {
```

Reply:
Sadly, this does not work here as `.internal` is defined as `canonicalType.has(Properties.IS_EXTERNAL -> false)`.
The canonical type is artificially created from the type alias and is external.

Reply:
oh, ok that's good to know then...